.github/workflows/test.yml

```diff
@@ -129,6 +129,8 @@ jobs:
           key: gradle-${{ hashFiles('*.gradle', 'gradle/dependency-locks/*') }}
       - name: Run Checkstyle
         run: ./gradlew checkstyleMain checkstyleTest --continue
+      - name: Run Spotless
+        run: ./gradlew spotlessCheck
```

Comment:
```suggestion
      - name: Run Style checks
        run: ./gradlew checkstyleMain checkstyleTest spotlessCheck --continue
```

Running them in one go seems simpler—the `--continue` makes sure that they all run even if one fails.

Reply:
Yup, works for me!